Upload test artifact/results

pyth0n1c opened the pull request:
Uploads the summary results to the s3
reporting bucket after a test completes.
This step was in place before but never
ran because 1 or more tests fail.

pyth0n1c committed:
Upload test artifact/results
after a scheduled test run.

dleung-splunk commented:
Might this PR get merged prior to the next release?  Just checking as I have some work that's dependent on the outcome of this, Thanks!  :)

pyth0n1c commented:
We have tagged it in preparation for the 3.56.0 release.  We expected to merge this tomorrow.

pyth0n1c committed:
Update detection-testing.yml

Accidentally removed change directory.
We were not in the correct directory
for the file we were trying to upload
to S3.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.
